runtime/src/accounts_db.rs

```diff
@@ -360,7 +360,7 @@ mod test_accounts_db_serialize {
     // These some what long test harness is required to freeze the ABI of
     // AccountsDB's serialization logic (AccontsDBSerialize) because
     // AccountsDBSerialize takes a ref(&) to an AccountsDB.
-    #[frozen_abi(digest = "8HonGUuN4veVMzn6JBsUd2o7AkTRkkrKxy1bQeB5rnvE")]
+    #[frozen_abi(digest = "EpX1HbzgTKh1jki8aVJKzTL3dWoA5cknxn6qr6WDrwm")]
```

Comment:
todo: this is a bit odd that actual serialized bytes is reduced so much: 572 => 180... Also, this ABI digest information is opaque due to known problem of custom serialization code (easily fixable, but introduces abi change...).

Reply:
Well, this is solved; snapshot_storages was empty.